src/routes/NavBar.svelte

```diff
@@ -12,60 +13,122 @@
     }
 </script>
 
-<nav class="flex items-center justify-center">
-    <ul
-        class="flex w-full flex-row items-center justify-between gap-4 bg-csi-neutral-50 p-1 text-csi-black shadow-md lg:w-fit lg:rounded-xl dark:bg-csi-neutral-950 dark:text-csi-white"
-    >
-        <li class="mx-6 sm:mx-10 lg:ml-8 lg:mr-4">
-            <img src={upcsi} alt="UP CSI Logo" class="h-8 object-contain" />
-        </li>
-        <li class="hidden lg:block"><a href="/">Home</a></li>
-        <li class="hidden lg:block"><a href="/about/">About</a></li>
-        <li class="hidden lg:block"><a href="/people/">People</a></li>
-        <li class="hidden lg:block"><a href="/events/">Events</a></li>
-        <li class="hidden lg:block"><a href="/projects/">Projects</a></li>
-        <li class="hidden lg:block">
-            <button class="flex items-center justify-center" on:click={theme.toggle}
-                ><Icon src={Moon} theme="solid" class="mx-2 h-6 dark:hidden" /></button
-            >
-            <button class="flex items-center justify-center" on:click={theme.toggle}
-                ><Icon src={Sun} theme="solid" class="mx-2 hidden h-6 dark:block" /></button
-            >
-        </li>
-        <li class="hidden lg:block">
-            <button class="rounded-lg bg-csi-blue px-5 py-2 text-csi-black">Get in Touch</button>
-        </li>
-        <li class="mx-6 my-2 flex flex-row items-center sm:mx-10 lg:hidden">
-            <button class="flex items-center justify-center" on:click={theme.toggle}
-                ><Icon src={Moon} theme="solid" class="mx-4 h-6 dark:hidden" /></button
-            >
-            <button class="flex items-center justify-center" on:click={theme.toggle}
-                ><Icon src={Sun} theme="solid" class="mx-4 hidden h-6 dark:block" /></button
-            >
-            <button
-                class="rounded-lg bg-black px-5 py-2 text-csi-white"
-                on:click={() => (mobileMenu = !mobileMenu)}>Get Started</button
-            >
-        </li>
-    </ul>
-</nav>
+<div
+    class="flex items-center justify-between rounded-full border-2 border-light-gray bg-csi-white/90 p-1 font-dm text-csi-black shadow-xl backdrop-blur-xl lg:mx-24 dark:border-dark-gray dark:bg-csi-black/90 dark:text-csi-white"
+>
```

Comment:
```suggestion
<div
    class="flex items-center justify-between rounded-full border-2 border-light-gray bg-csi-white/90 p-1 font-dm text-csi-black shadow-lg backdrop-blur-xl lg:mx-24 dark:border-dark-gray dark:bg-csi-black/90 dark:text-csi-white"
>
```
This is very minor, I just think that `shadow-xl` would be too much here especially since it already has a border. `shadow-lg` should suffice.